### DISCUSSION TITLE
Consistency of base language strings

### DISCUSSION THREAD

**Discussion by gujjwal00** [gujjwal00/avnc#40] [General]
AVNC was translated in few different languages recently. This effort has uncovered some minor inconsistencies in the writing style used for the base language English. While I have tried my best to be concise & clear about the strings used, three is still room for improvement. Its better to do this cleanup now, so that translators too can use a consistent style. 

I have looked through some design guidelines , and here is the summary:

- Title Case *vs* Sentence case: Sentence case seems to be the preferred choice on Android. We are currently using both, so, probably should switch to sentence case in all cases.
- Punctuation: Some of our short strings end with period(.) or exclamation(!). It is recommended to only use them for descriptive texts, so we should remove them.
- Some of the long strings can be simplified using active voice.
- Something else I missed?

This a half announcement/half feedback post, any suggestion is welcome.

cc @zequinh4, @eshagh79, @comradekingu

Resources:
https://material.io/design/communication/writing.html#principles
https://design.firefox.com/photon/copy/capitalization.html
https://design.firefox.com/photon/copy/punctuation.html

**Comment by comradekingu** [comment]
https://github.com/gujjwal00/avnc/pull/41 _App strings reworked_.
